settings.gradle.kts

```diff
@@ -15,9 +16,11 @@ pluginManagement {
     }
 }
 
-include(":plugin",
+include(
         ":samples:junit-tests",
         ":samples:kotlin-android",
         ":samples:kotlin-android-moshi-codegen",
         ":samples:kotlin-coroutines",
         ":samples:groovy-android")
+
+includeBuild("plugin-root")
```

Comment:
I ended up with `plugin-root` to make a distinction with the `plugin` module. Also if both project have the same name, it confuses the dependency substitution.

Reply:
Can I ask you to rename it to `gradle-plugin`? As `plugin` (or `plugin-root`) is less informative in this context?